src/api/Themer.ts

```diff
@@ -0,0 +1,83 @@
+import { Theme } from "../entities";
+import { Constants, getByName, setAMOLEDThemeEnabledBypass } from "../metro";
+import { readdir } from "../native/fs";
+import { Logger } from "../utils";
+import { THEME_DIRECTORY } from "../utils/constants";
+import { after } from "../utils/patcher";
+
+const logger = new Logger("Themer");
+export const themes = {} as Record<string, Theme>;
+export let currentTheme: Theme;
+
+enum ThemeType {
+    AMOLED = 2,
```

Comment:
dumb just reverse order and don't specify value it automatically counts up from zero

explode

Reply:
done